### PR TITLE
updated incorrect and outdated ports&IP table

### PR DESCRIPTION
**Pull request by MartinRinas**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-prep.md [hunk 1]
```diff
@@ -101,24 +101,8 @@ Microsoft Teams Rooms is designed to inherit Proxy settings from the Windows OS.
     
 To use this application, you must be able to connect to the endpoints described below. To see the IP addresses, expand the IP address section below the table describing the traffic flow.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-prep.md:102]:
```suggestion
See the [Network Security](https://docs.microsoft.com/microsoftteams/rooms/security#network-security) article for full details on FQDNs, ports, and IP address ranges required for Microsoft Teams Rooms.
```

File: Teams/rooms/rooms-prep.md [hunk 2]
```diff
@@ -101,24 +101,8 @@ Microsoft Teams Rooms is designed to inherit Proxy settings from the Windows OS.
...
-|Lync Mobile push notifications for Lync Mobile 2010 on iOS devices. You don't need this for Android, Nokia Symbian or Windows Phone mobile devices.  <br/> |Client Computer or Logged on user  <br/> |Ephemeral ports  <br/> |\*.contoso.com  <br/> |No  <br/> |Yes  <br/> |[Skype for Business IP ranges](https://support.office.com/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2?ui=en-US&amp;rs=en-US&amp;ad=US#BKMK_SfB_IP) <br/> |TCP 5223  <br/> |
-|Skype Telemetry  <br/> |Client Computer or Logged on user  <br/> |Ephemeral ports  <br/> |skypemaprdsitus.trafficmanager.net  <br/> pipe.skype.com  <br/> |No  <br/> |No  <br/> |N/A  <br/> |TCP 443  <br/> |
-|Skype client quick tips  <br/> |Client Computer or Logged on user  <br/> |Ephemeral ports  <br/> |quicktips.skypeforbusiness.com  <br/> |No  <br/> |No  <br/> |N/A  <br/> |TCP 443  <br/> |
-
-> [!NOTE]
-> The wildcard for contoso.com and broadcast.skype.com represents a long list of nodes that are exclusively used for Microsoft 365 or Office 365. 
+**Firewall Proxy Host Name/Port Configuration**
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-prep.md:104]:
```suggestion
```

File: Teams/rooms/rooms-prep.md [hunk 3]
```diff
@@ -101,24 +101,8 @@ Microsoft Teams Rooms is designed to inherit Proxy settings from the Windows OS.
...
+See this article for full details on required FQDNs, Ports and IPs to operate Microsoft Teams Rooms: https://docs.microsoft.com/en-us/microsoftteams/rooms/security#network-security
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-prep.md:105]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion only.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**